client/state/reader/conversations/selectors/is-following-reader-conversation.js

```diff
@@ -20,7 +15,7 @@ import 'state/reader/init';
  */
 export default function isFollowingReaderConversation( state, { siteId, postId } ) {
 	return (
-		get( state, [ 'reader', 'conversations', 'items', key( siteId, postId ) ] ) ===
+		state?.reader?.conversations?.items?.[ key( siteId, postId ) ] ===
```

Comment:
Assuming that `reader` reducer has been loaded, `conversations` and its `items` will always exist.  Also, what is the chance that we'll call the selector with a `null` or `undefined` `state`?